builtin/providers/openstack/resource_openstack_blockstorage_volume_attach_v2.go

```diff
@@ -0,0 +1,254 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/gophercloud/gophercloud/openstack/blockstorage/extensions/volumeactions"
+	"github.com/gophercloud/gophercloud/openstack/blockstorage/v2/volumes"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBlockStorageVolumeAttachV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBlockStorageVolumeAttachV2Create,
+		Read:   resourceBlockStorageVolumeAttachV2Read,
+		Update: nil,
```

Comment:
If everything is now ForceNew, we can drop this - the schema is more relaxed about this now :)